src/components/subscriptions/data/service.js

```diff
@@ -195,3 +112,41 @@ export function addUsers(options = {}) {
   return Promise.resolve(created);
   // return Promise.reject(new Error('Could not connect to the server'));
 }
+
+class LicenseManagerApiService {
+  static licenseManagerBaseUrl = `${configuration.LICENSE_MANAGER_BASE_URL}/api/v1/`;
+
+  static licenseAssign(options, subscriptionUUID) {
+    const url = `${LicenseManagerApiService.licenseManagerBaseUrl}subscriptions/${subscriptionUUID}/licenses/assign/`;
+    return apiClient.post(url, options, 'json');
+  }
+
+  static fetchSubscriptions(options) {
+    const queryParams = {
+      ...options,
+    };
+
+    const url = `${LicenseManagerApiService.licenseManagerBaseUrl}subscriptions/?${qs.stringify(queryParams)}`;
+    return apiClient.get(url);
+  }
+
+  static fetchSubscriptionUsers(subscriptionUUID, options) {
+    const queryParams = {
+      ...options,
+    };
+
+    const url = `${LicenseManagerApiService.licenseManagerBaseUrl}subscriptions/${subscriptionUUID}/licenses?${qs.stringify(queryParams)}`;
```

Comment:
Reiterating from my earlier comment that we should have trailing slash `/` at the end of these API URLs, e.g., `/licenses/?...`.